robottelo.properties.sample

```diff
@@ -72,3 +72,62 @@ driver=firefox
 # URL of the  exported data archive (typically a .tgz containing a bunch of CSV
 # files together with repo data)
 #export_tar.url=http://example.org/sat5_export_data.tgz
+
+# Section for performance tests parameters.
+[performance]
+# Control whether or not to time on hammer commands in robottelo/cli/base.py
+# Default set to be 0, i.e. no timing of performance is measured and thus no
+# interference to original robottelo tests.
+test.foreman.perf=0
+# Folowing entries are used for preparation of performance tests after a fresh install.
+# They will be used by test/foreman/performance/test_standard_prep.py, which supports:
+# 1) downloading manifest,
+# 2) uploading manifest to subscription,
+# 3) updating Red Hat CDN URL,
+# 4) enabling key repositories: rhel6-rpms, rhel7-rpms, rhel6-kickstart-rpms, rhel7-kickstart-rpms,
+#    rhel6-optional-rpms, rhel7-optional-rpms, rhel6-optional-source-rpms, rhel7-optional-source-rpms,
+#    rhel6-optional-debug-rpms, r7-optional-debug-rpms
+# Note that this preparation step is not required as long as satellite server is already configured.
+test.manifest.location=http://path/to/manifests/
+test.manifest.file=example-manifest-name.zip
+test.cdn.address=http://example.redhat.cdn/pub
+# Product-ID for enabling repositories by robottelo.cli.repository_set.RepositorySet
+test.pid=15
```

Comment:
Robottelo has tools that let you programmatically search for products via either CLI, API or UI. (Right now, the CLI is probably best suited for this task.) Just ask on IRC and we can help you.